### PR TITLE
Bug 1521886 - update thicknesss/color of ds-list horizontal rules

### PR DESCRIPTION
**Pull request by dmose**
How to test:

* go to about:config
* search for stream.config
* set that preference to {"enabled":true,"layout_endpoint":"https://getpocket.com/v3/newtab/layout?version=1&consumer_key=40249-e88c401e1b1f2242d9e441c4&layout_variant=dev-test-all"}
* open a new tab

Inspect the long horizontal rule at the top of the one of the list.  Also inspect the shorter line (which is implemented as a border) above one of the .ds-items.  Both should be 1 pixel thick, and the color of the lines should be #D7D7DB (which is the same as rgb(215, 215, 219)).

### CODE REVIEW

**Comment by Mardak** [comment]
Are you fixing the incorrect padding from the horizontal rules here?

**Comment by Mardak** [comment]
I guess you can fix the padding issues as part of bug 1521119 instead

File: content-src/components/DiscoveryStreamComponents/List/_List.scss [hunk 1]
```diff
@@ -3,7 +3,12 @@ $item-font-size: 13;
 $item-line-height: 20;
 
 .ds-list-border {
-  border: 1px solid $grey-40-36;
+  border: 1px solid $grey-30;
+  border-left: 0;
+  border-right: 0;
+  border-bottom: 0;
```
Comment (Mardak) [inline content-src/components/DiscoveryStreamComponents/List/_List.scss]: No need to reinvent the styling here with `$border-secondary` existing. Also you can reset all with just `border: 0`, so
```css
border: 0;
border-top: $border-secondary;
```

File: content-src/components/DiscoveryStreamComponents/List/_List.scss [hunk 2]
```diff
@@ -88,8 +93,8 @@ $item-line-height: 20;
 // Unfortunately the border needs to go _above_ the row gap as currently
 // set up, which means that some refactoring will be required to do this.
 .ds-list-item:nth-child(-n+3) { // all but the last three items
-  border-bottom: 2px solid $grey-40-36;
-  margin-bottom: -2px;  // cancel out the 2 pixels we used for the border
+  border-bottom: 1px solid $grey-30;
```
Comment (Mardak) [inline content-src/components/DiscoveryStreamComponents/List/_List.scss]: Same thing here: `border-bottom: $border-secondary;` Also grey-40-36 is no longer used and shouldn't exist, so remove it.

Comment (dmose) [reply content-src/components/DiscoveryStreamComponents/List/_List.scss]: Fixed.

File: content-src/components/DiscoveryStreamComponents/List/_List.scss (hunk 1 above)
Comment (dmose) [reply content-src/components/DiscoveryStreamComponents/List/_List.scss]: good points; fixed!

**Pull request merged**